docker-compose.yml

```diff
@@ -245,6 +255,7 @@ services:
     environment:
       MONGO_INITDB_ROOT_USERNAME: "prisma"
       MONGO_INITDB_ROOT_PASSWORD: "prisma"
+      INIT_WAIT_SEC: 10
```

Comment:
Nitpick: Can you try lowering it? Since this applies to all local devs, 10s is a long time to switch dev dbs.

Reply:
Until you hit problems again that is.